apps/vr-tests/src/stories/react-charting/AreaChart.stories.tsx

```diff
@@ -141,7 +141,7 @@ export const Basic = () => {
   return (
     <div style={rootStyle}>
       <AreaChart
-        culture={window.navigator.language}
+        culture="en-US"
```

Comment:
do we need same change in v9 stories also?

Reply:
I am not seeing these regressions in v9, so I am not sure if this is the correct fix.